### ISSUE TITLE
Remove Intel binary on universal apps on M1/Apple Silicon

### ISSUE THREAD

**Issue by SlCKB0Y** [IngmarStein/Monolingual#262]
Hi,

I was just submitting a feature request to allow the removal of "Intel" components from universal binaries when run on Apple Silicon Macs. 

Kind regards,
Stu.

**Comment by eugenesvk** [comment]
Would be great to have the option to delete either (Intel on Silicon and Silicon on Intel)

**Comment by kappesante** [comment]
totally

**Comment by JohnnyFireOne** [comment]
im in to this!!!!!!

**Comment by MaurizioLoreti** [comment]
Agreed.  I configure Monolingual to remove all from universal binaries except ARM and, the next time I start Monolingual, it reverts to "Delete all except Intel".  EXTREMELY annoying.
